spdx-utils/build.gradle.kts

```diff
@@ -72,7 +74,7 @@ val importSpdxLicenseTexts by tasks.registering(SvnExport::class) {
     description = "Imports license texts from the SPDX repository."
     group = "SPDX"
 
-    svnUrl = "https://github.com/spdx/license-list-data/trunk/text"
+    svnUrl = "https://github.com/spdx/license-list-data/z/text"
```

Comment:
What's the meaning of this change?

Reply:
This change was not intended. Removed it.